WeakAuras/Types.lua

```diff
@@ -1440,6 +1446,33 @@ WeakAuras.bufftrigger_progress_behavior_types = {
   showAlways= L["Always"]
 }
 
+WeakAuras.bufftrigger_2_progress_behavior_types = {
+  showOnActive = L["Aura(s) Found"],
+  showOnMissing = L["Aura(s) Missing"],
+  showAlways= L["Always"]
+}
+
+WeakAuras.bufftrigger_2_preferred_match_types =
+{
```

Comment:
opening brace here is on a newline, which is inconsistent with your other additions (and the rest of the file)